index.js

```diff
@@ -667,43 +673,43 @@ class Plugin extends MiniAccountsPlugin {
         Buffer.from(account.getPaychan().publicKey.substring(2), 'hex')
       )
     } catch (err) {
-      debug('verifying signature failed:', err.message)
+      this._log.debug('verifying signature failed:', err.message)
     }
 
     // TODO: better reconciliation if claims are invalid
     if (!valid) {
-      debug(`got invalid claim signature ${signature} for amount ${dropAmount} drops`)
+      this._log.debug(`got invalid claim signature ${signature} for amount ${dropAmount} drops`)
       /* throw new Error('got invalid claim signature ' +
         signature + ' for amount ' + amount + ' drops') */
       throw new Error('Invalid claim: invalid signature')
     }
 
     // validate claim against balance
     const channelBalance = util.xrpToDrops(account.getPaychan().amount)
-    debug('got channel balance. balance=' + channelBalance)
+    this._log.trace('got channel balance. balance=' + channelBalance)
     if (new BigNumber(dropAmount).gt(channelBalance)) {
       const message = 'got claim for amount higher than channel balance. amount: ' + dropAmount + ', incoming channel balance: ' + channelBalance
-      debug(message)
+      this._log.debug(message)
```

Comment:
I'd call this an error, because it's pretty rare and suggests someone is trying to steal from us (or else there's been some crazy error with channel configuration)